packages/location_permissions/android/src/main/java/com/baseflow/location_permissions/LocationPermissionsPlugin.java

```diff
@@ -139,6 +167,15 @@ public void onMethodCall(MethodCall call, Result result) {
     }
   }
 
+  @Override
+  public void onListen(Object arguments, EventSink events) {
+    eventSink = events;
+  }
+
+  @Override
+  public void onCancel(Object arguments) {
```

Comment:
Shouldn't we remove our self as a registered receiver from the active context here? And maybe clean up the `eventSink` (e.g. set to `null`)?

Reply:
Setting the field to `null` now.
Unregistering the receiver did not make sense without moving the registry part as well (because there wouldn't be a chance to re-register again with the current implementation), so I moved that part too.

Reply:
Any more remarks here?